rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -658,6 +658,7 @@ public FileInfo load(String key) throws Exception
       avail += new File(CacheUtil.getDirPath(d, conf)).getUsableSpace();
     }
     avail = BYTES.toMB(avail);
+    avail -= DiskUtils.getCacheSizeMB(conf);
```

Comment:
should we add check in case `avail`comes out to be 0 and then we decrease the value, making it a negative number?